bindings/java/src/main/native/Makefile.am

```diff
@@ -103,22 +103,32 @@ set-version:
 repack-jar: $(jarfile)
 	$(RM) -r $(java_build_dir)/repack_dir
 	unzip -o $(jarfile) -d $(java_build_dir)/repack_dir
-	\cp -v $(topdir)/libjucx_*.so/libjucx_*.so $(java_build_dir)/repack_dir
+	\cp $(topdir)/libjucx_amd64.so/libjucx_amd64.so $(topdir)/bindings/java/resources
```

Comment:
1. why need to copy both to resources and repack_dir?
2. can we copy with wildcard? anyway we have test step to make sure the files are present

Reply:
1. The resources dir is used by Maven while rebuilding Jars. The repack_dir is used while repacking. This is a temporary solution before we externalize both repack and check steps to CI.
2. Changed to wildcards.